catroid/build.gradle

```diff
@@ -101,7 +117,7 @@ android {
             buildConfigField "String", "PROJECT_NAME", "\"${(String) getProjectName()}\"";
             buildConfigField "String", "START_PROJECT", "\"${(String) getProgramId(true) }\""
             buildConfigField "boolean", "FEATURE_APK_GENERATOR_ENABLED", "true"
-            buildConfigField "boolean", "FIREBASE_CRASH_REPORT_ENABLED", "false"
+            buildConfigField "boolean", "CRASHLYTICS_CRASH_REPORT_ENABLED", "true"
```

Comment:
My fault. I meant don't change it. Maybe one day we will enable it for standalone as well, but
![not today](http://i.imgur.com/dMin28b.jpg?1)